hs/Css/DrawCss.hs

```diff
@@ -40,53 +37,45 @@ mainCSS = "#main" ? do
     "border-radius" -: "8px"
     border solid (px 2) black
 
-{- The grid background for SVG canvas. -}
-canvasBackground = "#background" ? do
-    height100
-    width100
-    "background-image" -: "url(/static/res/grid.gif)"
-    "background-size" -: "128px"
-    opacity 0.3
-
-{- The SVG canvas. -}
-canvasCSS = "#mySVG" ? do
-    height100
-    width100
-    position absolute
-    top nil
-    left nil
+{- The SVG canvas and the grid background. -}
+canvasCSS = do
+    "#background" ? do
+        height100
+        width100
+        "background-image" -: "url(/static/res/grid.gif)"
+        "background-size" -: "128px"
+        opacity 0.3
+    "#mySVG" ? do
```

Comment:
Hmm. We haven't been using camel case in IDs, and we haven't been using the `my` prefix. I realize that this name isn't new. I'll make a note of it to change in the future. There are a few names that do this.